.circleci/config.yml

```diff
@@ -22,6 +22,10 @@ jobs:
             fi
             if [ -f "$BASH_ENV" ]; then source $BASH_ENV; fi
             $SUDO SINGULARITYENV_OPHYS_ETL_TAG=${CIRCLE_BRANCH} singularity build suite2p_container.img src/ophys_etl/transforms/suite2p_container.def
+      - run:
```

Comment:
There's 85 replies on our slack thread so I'll just ask -- could the singularity behavior be a separate PR or is it a dependency? Seems like there's kind of 2 features here: the singularity changes and the filtering.